src/domain.rs

```diff
@@ -1,12 +1,14 @@
-use crate::sync::atomic::{AtomicIsize, AtomicPtr, AtomicU64, AtomicUsize};
+use crate::sync::atomic::{AtomicIsize, AtomicPtr, AtomicUsize};
 use crate::{Deleter, HazPtrRecord, Reclaim};
 use alloc::boxed::Box;
 use alloc::collections::BTreeSet;
 use core::marker::PhantomData;
 use core::sync::atomic::Ordering;
 
-#[cfg(feature = "std")]
+#[cfg(all(feature = "std", not(miri)))]
```

Comment:
I'm not sure I follow why it's necessary to turn this off under miri?

Reply:
It goes with the other "time" related things so I believe it makes things cleaner when `due_time` is disabled (see comment below)

Reply:
Hmm, I still don't understand after reading your comment. From what I can tell, miri already mocks time just fine (https://github.com/rust-lang/miri/pull/975), and supports `Instant::now` (https://github.com/rust-lang/miri/pull/1243), and I don't see anything about time-related limitations in the miri readme?

Reply:
Actually, I'm going to merge this as-is just so we can iterate _just_ on the miri point and have working CI in the meantime!

Reply:
Ah, I was getting the `clock_gettime` not available when isolation is enabled error, not knowing about `-Zmiri-disable-isolation`. This should be pretty easy to address in a PR.